.eslintrc.js

```diff
@@ -1,6 +1,11 @@
 module.exports = {
   "extends": "airbnb-base",
   "env": {
-    "jest": true
+    "jest": true,
+  },
+  rules: {
+    "no-console": "off",

```

Comment:
Question: why `"off"` vs. `0`, which you use below?

Also, since this is a `.js` file, we can include comments, so would you mind adding a comment above each of these new rules explaining what you're doing here?  That way others can discover what this is all about.

Reply:
For `no-console`, I used what eslint uses in their examples [here](https://eslint.org/docs/rules/no-console), and when I was testing `func-names` lint complained and wanted me to use 0. I can test if `no-console` works with `0` or if `func-names` works with `off` to have more consistency.

Also, I didn't add any comments because all the info to read about these changes is included in the description of the PR. Should I still add it to the js file?

Reply:
I changed it and now they all use `off`.